app/controllers/admin/challenges_controller.rb

```diff
@@ -40,13 +41,33 @@ def update
 
     private
 
+    def list_challenges
+      if params[:search].present?
+        Repo::Challenge.where('challenges.title LIKE ?', "%#{params[:search]}%")
```

Comment:
risk of SQL injection?

Reply:
Thanks. I will add sanitizer within next MR